src/dsp_tools/resources/start-stack/docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
   app:
     # on the verge of every deployment (fortnightly), update the "image" value from the "app" value of
     # https://github.com/dasch-swiss/ops-deploy/blob/main/roles/dsp-deploy/files/RELEASE.json
-    image: daschswiss/dsp-app:v11.3.4
+    image: daschswiss/dsp-app:v11.5.1
```

Comment:
https://github.com/dasch-swiss/ops-deploy/blob/main/roles/dsp-deploy/files/RELEASE.json#L2C34-L2C50 has 11.6.1. Is this on purpose?

Reply:
an hour ago it was correct. I was going to check all the numbers tomorrow morning again but didn't want to have to wait for an approve tomorrow...